creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -86,6 +87,7 @@ const updateReplicaSetJobProcessor = async function ({
   healthyNodes = await getCachedHealthyNodes()
   if (healthyNodes.length === 0) {
     try {
+      tracing.info('init AudiusLibs')
```

Comment:
whats your thinking around these tracing logs? should we be replacing our other logs with these

Reply:
I think we should replace our current logging setup with tracing logs - I think it's a separate discussion though and I wanna get the whole team's input on it (which'll require everyone to get used to working with tracing and CloudTrace)


For now, the tracing logs in our code are there to help me debug and to test out if it's something useful